### DISCUSSION TITLE
Show RCMs on stream as a popup

### DISCUSSION THREAD

**Discussion by Petarkco** [f1multiviewer/issue-tracker#173] [Feature requests]
Add an option to show RCMs on a stream in a window with a transparent background (similar to screenshot)
![image](https://user-images.githubusercontent.com/25685344/187163683-ce006140-99e9-4754-b92a-16ac52d8ad9d.png)

Have options to change size, how long it appears for etc.

**Comment by LapsTimeOFF** [comment]
It's a good idea, but they are already show on the top of the stream

**Comment by jeppeolesen** [comment]
Closing this as they are already included in the main broadcast, and there would be little use for it on the onboards.
